org.pjbroad.EternallandsClient.appdata.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+	<id>org.pjbroad.EternallandsClient</id>
+	<name>Eternal Lands</name>
+	<developer_name>Paul Broadhead</developer_name>
+	<summary>Eternal Lands is a free to play, graphical MMORPG</summary>
+	<description>
+		<p>
+	Eternal Lands is a free to play, graphical MMORPG (massively-multi-player
+	online role-playing game).  Different from other role-playing games, there are
+	no fixed classes so you can improve any of the many skills.  There is a choice
+	of several character races but all are equal.
+		</p>
+	</description>
+	<releases>
+		<release version="1.9.5p2" date="2018-06-29"/>
+	</releases>
+	<screenshots>
+		<screenshot>
+			<image type="source">http://www.eternal-lands.com/forum/uploads/gallery/category_5/med_gallery_4_5_804654.png</image>
+		</screenshot>
+		<screenshot>
+			<image type="source">http://www.eternal-lands.com/forum/uploads/1218074388/med_gallery_4_16_52708.jpg</image>
+		</screenshot>
+		<screenshot>
+			<image type="source">http://www.eternal-lands.com/forum/uploads/1218074388/med_gallery_4_16_4336.jpg</image>
+		</screenshot>
+	</screenshots>
+	<metadata_license>CC0-1.0</metadata_license>
+	<project_license>QPL-1.0 LicenseRef-Propritary</project_license>
```

Comment:
AFAIK there should be `and` between two licenses.

Reply:
By the way, I just noticed a typo: *LicenseRef-Propritary* should be probably *LicenseRef-proprietary*. ;-)

Reply:
OK, thanks.